Octokit.Reactive/Clients/IObservableRepositoryCommitsClients.cs

```diff
@@ -40,11 +40,28 @@ public interface IObservableRepositoryCommitsClient
         /// </summary>
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
+        /// <param name="options">Options for changing the API response</param>
```

Comment:
We're close to getting this in, so I think I'm less worried about making this change here...